community-operators/etcd/etcdoperator.clusterserviceversion.yaml

```diff
@@ -3,6 +3,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  description: etcd is a distributed key value store providing a reliable way to store data across a cluster of machines.
```

Comment:
Was this supposed to be moved to `metadata.annotations`

Reply:
yes, fixed